main.css

```diff
@@ -0,0 +1,148 @@
+
+.head-triangle {
+    position: relative;
+    top: 10px;
+    left: 10px;
+    width: 0;
+    height: 0;
+    border-left: 300px solid transparent;
```

Comment:
Instead of using border tricks, the roof can also be done with a rectangle with `transform: rotate(45deg)`. Would recommend having a play with it too, as transforms will become very useful as you build more advanced applications!

[What are CSS transforms and how to use them >](https://css-tricks.com/almanac/properties/t/transform/)